### PR TITLE
adds logic that sets a jira project properties on project provisionin…

### LINKED ISSUE #452: setup webhook in jira projects on project creation event
**Is your feature request related to a problem? Please describe.**
To integrate Jira to be able to trigger Jenkins jobs we need to configure a webhook in the jira project at creation time.

### PR DESCRIPTION
**Pull request by stitakis**
fixes #452

### CODE REVIEW

**Review by martsec — APPROVED** [review]

File: src/main/java/org/opendevstack/provision/services/JiraAdapter.java [hunk 1]
```diff
@@ -453,6 +443,30 @@ public OpenProjectData createBugtrackerProjectForODSProject(OpenProjectData proj
...
+    if (!Boolean.valueOf(addProjectProperty)) {
+      logger.debug(
+          "Project type is not configured for the setup of webhook proxy url as jira project property [projectKey={}, projectType={}]",
+          projectKey,
+          projectType);
+      return;
+    }
+
+    logger.info(
+        "Project type is not configured for the setup of webhook proxy url as jira project property [projectKey={}, projectType={}]",
```
Comment (georgfedermann) [inline src/main/java/org/opendevstack/provision/services/JiraAdapter.java:462]: Isn't this what was checked in 453?

Comment (stitakis) [reply src/main/java/org/opendevstack/provision/services/JiraAdapter.java:462]: good spotted... actually the intention here is log a message but the message is wrong... I'll fix it

Comment (stitakis) [reply src/main/java/org/opendevstack/provision/services/JiraAdapter.java]: remove the word `not` and pushed the change

**Comment by clemensutschig** [comment]
Build failed..

**Comment by stitakis** [comment]
@clemensutschig the java compilation and unit test passed. The error was raised in the build docker image stage with this message: 
```
-> Run COMMIT_AUTHOR=$(git --no-pager show -s --format='%an (%ae)' $GITHUB_SHA)
fatal: bad object 94a87e4f6ad38d7354fa3c49769192429b4344da
```
This error is popping up since a couple of weeks randomly.
I'm investigating this but I was not able to do much progress atm.

**Review by georgfedermann — APPROVED** [review]

**Pull request merged**